lib/Parse/ParseDecl.cpp

```diff
@@ -6655,46 +6703,53 @@ Parser::parseDeclPrecedenceGroup(ParseDeclOptions flags,
         diagnose(Tok, diag::expected_precedencegroup_assignment);
         return abortBody();
       }
+      skipPostfixCodeCompletionToken(*lastTokTextEnd);
       continue;
     }
 
-    bool isLowerThan = false;
-    if (attrName == "higherThan" ||
-        (isLowerThan = (attrName == "lowerThan"))) {
+    if (attrName == "higherThan" || attrName == "lowerThan") {
```

Comment:
Is there a reason to change this?  Moving this code adds an extra string comparison (admittedly it might not matter, but it doesn't seem related to this patch).